bower.json

```diff
@@ -6,7 +6,7 @@
     "o-grid": "^4.2.0",
     "o-icons": ">=4.4.2 <6",
     "o-typography": "^4.1.0",
-    "o-visual-effects": "^0.1.0",
+    "o-visual-effects": "^1.0.0",
```

Comment:
Can you make this a range instead? Otherwise this release to o-header will need to be a major version so as not to break any build service builds